separate storage on QField and QFC (#393)
opengisch/QField-docs: documentation/how-to/projects.en.md

QField has a file selector that allows to open a project from the device locally. To open files from the cloud see [QFieldCloud ](../get-started/tutorials/get-started-qfc.md).

!!! note

>     Starting with Android 11 apps are denied full access to main and external storage content. For QField, this means direct access to projects and datasets transferred and/or downloaded into storage folders is not possible anymore. You can know more [on this page](../get-started/storage-qf.md).

Reviewer: ```suggestion
    Starting with Android 11 apps are denied full access to main and external storage content. For QField, this means direct access to projects and datasets transferred and/or downloaded into storage folders is not possible anymore. You can know more about [QFiled storage access](../get-started/storage-qf.md).
```

Reviewer 2: s/QFiled/QField

Reviewer: Thanks for spotting, fixed in the comment above.
